### PR TITLE
Add perplexity extension

### PR DESCRIPTION
**Pull request by third774**
## Description

A simple plugin for querying Perplexity from within Raycast.

## Screencast

https://github.com/raycast/extensions/assets/8732191/90366f50-f85f-4db0-a526-0555f820431b

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/perplexity/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  return (
+    <Form
+      actions={
+        <ActionPanel>
+          <Action.SubmitForm onSubmit={handleSubmit} />
+        </ActionPanel>
+      }
+    >
+      <Form.Description text="Where knowledge begins" />
+      <Form.TextArea id="query" title="Ask anything..." autoFocus />
```
Comment (pernielsentikaer) [inline extensions/perplexity/src/index.tsx:29]:
```suggestion
      <Form.TextArea id="query" title="Ask Anything..." autoFocus />
```

File: extensions/perplexity/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+type Values = {
+  query: string;
+  copilot: boolean;
+  focus: string;
+};
+
+export default function Command() {
+  function handleSubmit({ query: q, copilot, focus }: Values) {
+    const params = new URLSearchParams({
```
Comment (pernielsentikaer) [inline extensions/perplexity/src/index.tsx:11]: You might want to validate that query is not empty and show a failure tost here

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/perplexity/src/index.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      >
+        <Form.Dropdown.Item value="all" title="All" />
+        <Form.Dropdown.Item value="scholar" title="Academic" />
+        <Form.Dropdown.Item value="writing" title="Writing" />
+        <Form.Dropdown.Item value="wolfram" title="Wolfram Alpha" />
+        <Form.Dropdown.Item value="youtube" title="Youtube" />
+        <Form.Dropdown.Item value="reddit" title="Reddit" />
+      </Form.Dropdown>
+      <Form.Separator />
+      <Form.Checkbox defaultValue={false} id="copilot" label="Co-pilot" storeValue />
```
Comment (pernielsentikaer) [inline extensions/perplexity/src/index.tsx:43]:
```suggestion
      <Form.Checkbox defaultValue={false} id="copilot" label="Copilot" storeValue />
```

File: extensions/perplexity/src/index.tsx (hunk 2 above)
Comment (third774) [reply extensions/perplexity/src/index.tsx:18]: Ah, good call! Went with an error message instead to put the feedback closer to the input.

**Comment by third774** [comment]
Done pushing changes, and ready for another review! I addressed the feedback, updated the logo and screenshot, and added draft functionality. 🙏🏻

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**